build/Build.cs

```diff
@@ -152,8 +153,25 @@ public Build()
                 using var versionInfoFile = ModifyTemplatedVersionAndProductFilesWithValues();
                 using var productWxsFile = UpdateMsiProductVersion();
 
-                RuntimeIds.Where(x => x.StartsWith("win"))
-                    .ForEach(runtimeId => RunBuildFor(runtimeId.Equals("win") ? NetFramework : NetCore, runtimeId));
+                var runtimeIds = RuntimeIds.Where(x => x.StartsWith("win"));
+                
+                foreach (var runtimeId in runtimeIds)
```

Comment:
this looks like duplicate of the other switch the looks the same, should they be in sync? if so maybe a refactor this out?